### PR TITLE
Update inputs to new theming style

### PR DESCRIPTION
**Pull request by colinlohner**
## Overview
Update inputs to new theming style

## Screenshots (if applicable)
n/a

## Testing

- [ ] Updated Typescript types and/or component PropTypes
- [ ] Added / modified component docs
- [ ] Added / modified Storybook stories

### CODE REVIEW

File: src/theme/src/default-styles/inputs.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
+export default function getDefaultStyles(theme) {
```
Comment (mshwery) [inline src/theme/src/default-styles/inputs.js:1]: What are "default" styles? Is this meant to be part of the v6 theme ?

File: src/theme/src/hooks/useInputAppearance.js [hunk 2]
```diff
@@ -17,7 +18,9 @@ const hoverPlaceholderState = '&:hover::placeholder'
 const focusPlaceholderState = '&:focus::placeholder'
 
 const getInputStyles = (theme, appearance) => {
-  const { inputs } = theme
+  const { inputs: themeStyles } = theme
+  const defaultStyles = getDefaultStyles(theme)
+  const inputStyles = { ...defaultStyles, ...themeStyles }
```
Comment (mshwery) [inline src/theme/src/hooks/useInputAppearance.js:23]: Hm... IMO we should either:
1. explicitly tell consumers that if they don't pass in a complete theme object we will merge it with the v6 theme
2. provide a `createTheme` method for users to pass partial theme objects

File: src/theme/src/hooks/useInputAppearance.js [hunk 3]
```diff
@@ -17,7 +18,9 @@ const hoverPlaceholderState = '&:hover::placeholder'
 const focusPlaceholderState = '&:focus::placeholder'
 
 const getInputStyles = (theme, appearance) => {
-  const { inputs } = theme
+  const { inputs: themeStyles } = theme
+  const defaultStyles = getDefaultStyles(theme)
```
Comment (mshwery) [inline src/theme/src/hooks/useInputAppearance.js:22]: Can we instead pull this into the themes that rely on it (e.g. v5 and v6)? All styles should come from the theme itself, not from imported modules.

**Comment by colinlohner** [comment]
Closing this, since we are no longer going to do this approach

**Pull request closed without merging**